inference-engine/ie_bridges/java/samples/face_detection_sample_async/Main.java

```diff
@@ -99,137 +96,149 @@ public static void main(String[] args) {
+
+                                            startedRequestsIds.add(i);
+                                            request.StartAsync();
+                                        }
+                                    }
+                                    processInferRequets(WaitMode.RESULT_READY);
+                                } catch (InterruptedException e) {
+                                    e.printStackTrace();
+
+                                    for (Thread t : Thread.getAllStackTraces().keySet())
+                                        if (t.getState() == Thread.State.RUNNABLE) t.interrupt();
+                                }
+                            }
+                        });
 
         captureThread.start();
         inferThread.start();
 
-        TickMeter tm = new TickMeter();       
+        TickMeter tm = new TickMeter();
+        Scalar color = new Scalar(0, 255, 0);
         try {
             while (inferThread.isAlive() || !detectionOutput.isEmpty()) {
 
-                float[] detection = detectionOutput.poll(waitingTime, TimeUnit.SECONDS);  
-                if (detection == null)
-                  continue;
-                                
-                Mat img = processedFramesQueue.poll(waitingTime, TimeUnit.SECONDS); 
+                float[] detection = detectionOutput.poll(waitingTime, TimeUnit.SECONDS);
+                if (detection == null) continue;
+
+                Mat img = processedFramesQueue.poll(waitingTime, TimeUnit.SECONDS);
                 int maxProposalCount = detection.length / 7;
 
                 for (int curProposal = 0; curProposal < maxProposalCount; curProposal++) {
                     int imageId = (int) detection[curProposal * 7];
-                    if (imageId < 0)
-                        break;
-        
+                    if (imageId < 0) break;
+
                     float confidence = detection[curProposal * 7 + 2];
 
                     // Drawing only objects with >70% probability
-                    if (confidence < CONFIDENCE_THRESHOLD)
-                        continue;
-                    
+                    if (confidence < CONFIDENCE_THRESHOLD) continue;
+
                     int label = (int) (detection[curProposal * 7 + 1]);
                     int xmin = (int) (detection[curProposal * 7 + 3] * img.cols());
                     int ymin = (int) (detection[curProposal * 7 + 4] * img.rows());
                     int xmax = (int) (detection[curProposal * 7 + 5] * img.cols());
                     int ymax = (int) (detection[curProposal * 7 + 6] * img.rows());
-                    
+
                     // Draw rectangle around detected object.
-                    Imgproc.rectangle(img, new Point(xmin, ymin), new Point(xmax, ymax), new Scalar(0, 255, 0), 2); 
+                    Point min = new Point(xmin, ymin);
+                    Point max = new Point(xmax, ymax);
```

Comment:
what if
```java
Point tl = new Point((int) (detection[curProposal * 7 + 3] * img.cols()),
                     (int) (detection[curProposal * 7 + 4] * img.rows()));
Point br = new Point((int) (detection[curProposal * 7 + 5] * img.cols()),
                     (int) (detection[curProposal * 7 + 6] * img.rows()));

```

Reply:
It will be:
```java
Point tl =
         new Point(
                 (int) (detection[curProposal * 7 + 3] * img.cols()),
                 (int) (detection[curProposal * 7 + 4] * img.rows()));
Point br =
         new Point(
                 (int) (detection[curProposal * 7 + 5] * img.cols()),
                 (int) (detection[curProposal * 7 + 6] * img.rows()));
```